clusterloader/cluster_loader.go

```diff
@@ -154,34 +164,26 @@ func createTemplate(baseName string, ns *v1.Namespace, configPath string, numObj
 
 		// If there is a tuning set defined for this template
 		if tuning != nil {
-			if tuning.Templates.RateLimit.Delay != 0 {
-				framework.Logf("Sleeping %d ms between template creation.", tuning.Templates.RateLimit.Delay)
-				time.Sleep(time.Duration(tuning.Templates.RateLimit.Delay) * time.Millisecond)
+			if tuning.Templates.RateLimit.Delay != "" {
+				duration, err := clusterloaderframework.TimeSplit(tuning.Templates.RateLimit.Delay)
```

Comment:
Instead of introducing a new custom function, you can just use:
https://golang.org/pkg/time/#ParseDuration